README.md

```diff
@@ -0,0 +1,49 @@
+# Integreat Compass
+
+This system gathers courses that job center employees can pass on in counseling sessions to refugees and newcomers.
+
+## TL;DR
+
+### Prerequisites
+
+Following packages are required before installing the project (install them with your package manager):
+
+* `npm` version 7 or later
+* `nodejs` version 12 or later
+* `python3` version 3.9 or later
+* `python3-pip` (Debian-based distributions) / `python-pip` (Arch-based distributions)
+* `python3-venv` (only on Debian-based distributions)
+* Either `postgresql` **or** `docker` to run a local database server
```

Comment:
```suggestion
```
Since you proposed sqlite in #3, I think the dev setup does not need a separate db technology?